code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -653,18 +653,19 @@
 	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Fix the ship, find a way to power it and follow your captain's orders."
 	important_info = "Obey orders given by your captain. DO NOT let the ship fall into enemy hands."
 	outfit = /datum/outfit/syndicatespace/syndicrew
+	assignedrole = "Cybersun Crewmember"
+
+/obj/effect/mob_spawn/human/syndicatespace/Initialize(mapload)
+	. = ..()
+	var/policy = get_policy(ROLE_SYNDICATE_CYBERSUN)
+	important_info = policy || important_info
```

Comment:
Won't this return 1 since you're asking `if policy or important_exists` rather than `set it to policy if it exists, else use important_info`?

Reply:
It should use standard info if there is no policy(aka, policy is null)

Reply:
Okay, used "if(policy)" instead.